### PR TITLE
resolve ambiguous source field

### PR DESCRIPTION
**Pull request by FynnBe**
rename to rdf_source where applicable

### CODE REVIEW

**Comment by oeway** [comment]
Looks good! Maybe check if the website is normal after merging this PR.

File: scripts/imjoy_plugin_parser.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def parse_imjoy_plugin(source, overwrite_config=None):
     return config
 
 
-def convert_config_to_rdf(plugin_config, plugin_id, source_url=None):
+def convert_config_to_rdf(plugin_config, plugin_id, source_url=None):  # todo: remove special source treatment now that we use rdf_source
```
Comment (oeway) [inline scripts/imjoy_plugin_parser.py:118]: I think here we don't need any fix for the source, it's a source key to the source of the application (*.imjoy.html), not the rdf.

**Comment by oeway** [comment]
@FynnBe any idea on why ci breaks with main branch?

**Comment by FynnBe** [comment]
>@FynnBe any idea on why ci breaks with main branch?

one renaming was missing: https://github.com/bioimage-io/collection-bioimage-io/commit/85b3442f99b3b2c2b82c5dea25de8df80c1c425e

**Pull request merged**